exams/tasks_test.py

```diff
@@ -58,9 +60,40 @@ def test_task_retry(self, task, task_retry):
             task.delay()
 
         retry.assert_called_once_with(countdown=1, exc=error)
-        return self
+
+    @data(
+        (
+            export_exam_authorizations,
+            'export_exam_authorizations is improperly configured, please review require settings.'
+        ),
+        (
+            export_exam_profiles,
+            'export_exam_profiles is improperly configured, please review require settings.'
+        )
+    )
+    @unpack
+    def test_task_improperly_config_logged(self, task, expected_warning_message):
+        """
+        Verify that when a ImproperlyConfigured error occurs that the task logs exception
+        """
+        error = ImproperlyConfigured()
```

Comment:
Tiny nit: no need for the local here, could inline to the assignment on line 81.